FreeRTOS-Plus/Demo/FreeRTOS_Plus_TCP_UDP_Mode_CLI_Windows_Simulator/WIN32.vcxproj.user

```diff
@@ -1,3 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-</Project>
```

Comment:
This file seems like an outlier. Just making sure that it was intended.

Reply:
ya its a left over file from someone running the demo. it shouldn't be there and is automatically generated by visual studio when you open the solution.